keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -73,14 +74,38 @@ class SqlResourceRepository(
       } ?: throw NoSuchResourceName(name)
   }
 
+  override fun hasManagedResources(application: String): Boolean {
+    jooq
+      .select(RESOURCE.NAME)
+      .from(RESOURCE)
+      .where(RESOURCE.APPLICATION.eq(application))
+      .fetchArray(RESOURCE.NAME)
+      .let { results ->
+        if (results.isNotEmpty()) {
+          return true
+        }
+      }
+    return false
```

Comment:
Think you can just do `jooq.selectOne().from…` rather than having to filter results.

Reply:
I tried this, and if you say `jooq.selectOne()` and there is more than one resource it throws an error. The normal `.fetch()` always returns a record and therefor is never null. What I'm doing here definitely doesn't seem great though. I'll keep thinking

Reply:
I found a better way thanks to Zed! `selectCount()`